package.json

```diff
@@ -56,6 +56,7 @@
     "broccoli-asset-rev": "^2.4.5",
     "ember-array-helper": "^2.1.0",
     "ember-cli": "~3.1.4",
+    "ember-cli-code-coverage": "1.0.0-beta.4",
```

Comment:
Would maybe have been better to add this under a separate PR... but good to have anyway, thanks! 😉 🤷‍♂️

Reply:
For traceability's sake it's def better to do this separately since it's nothing to do with adding the dropzone component 😛